wiki/Beatmap/Category/it.md

```diff
@@ -0,0 +1,50 @@
+# Categorie di beatmap
+
+Le [beatmaps](/wiki/Beatmap) caricate online sono suddivise in categorie in base alla loro recente attività , [nominazioni](/wiki/Beatmap_ranking_procedure#qualification) stato e popolarità.
```

Comment:
It's nomination status not nomination, status ecc... so it should be translated to something like this or siminar to this

```suggestion
Le [beatmaps](/wiki/Beatmap) caricate online sono suddivise in categorie in base alla loro recente attività, stato della [nominazione](/wiki/Beatmap_ranking_procedure#qualification) e popolarità.
```